incubator/hnc/internal/reconcilers/anchor.go

```diff
@@ -130,26 +130,26 @@ func (r *AnchorReconciler) onDeleting(ctx context.Context, log logr.Logger, inst
 	// means HNC is being uninstalled and we shouldn't delete _any_ namespaces.
 	deletingCRD, err := isDeletingCRD(ctx, r, api.Anchors)
 	if err != nil {
-		log.Info("Couldn't determine if CRD is being deleted")
+		log.Error(err, "Couldn't determine if CRD is being deleted")
 		return false, err
 	}
-	log.Info("Anchor is being deleted", "deletingCRD", deletingCRD)
+	log.V(1).Info("Anchor is being deleted", "deletingCRD", deletingCRD)
```

Comment:
Why this is `.V(1)`? Do you mean that we don't need log for anchor being deleted or there's another log later when the anchor is really deleted?

Reply:
Yup, the "Anchor deleted" message is on line 70. This line is usually printed out 2-3 times - once when the user starts to delete the anchor ("deleting subnamespace"), often a second time when the subnamespace starts getting deleted ("waiting for subnamespace to be fully purged"), and a third time when the finalizers are removed. But that's just an artifact of how the reconciler works and isn't useful to a cluster admin, which is why I made it V(1).

Reply:
sg